Documentation/FrameworkOverview.md

```diff
@@ -47,10 +44,37 @@ Signals send three different types of events to their subscribers:
  * The **completed** event indicates that the signal finished successfully, and
    that no more values will be added to the stream. Completion must be handled
    specially – it is not included in the stream of values.
+ * The **interrupted** event indicates that the signal has terminated 
+   non-erroneous, yet unsuccessful e.g. when the corresponding request has been
+   cancelled before the signal could finish. Interruptions must be handeled 
+   specially - they are not included in the stream's values.
 
 The lifetime of a signal consists of any number of `next` events, followed by
-one `error` or `completed` event (but not both).
+one `error`, `completed` or `interrupted` event (but no combination of those).
 
+
+### Signal Producers
+
+A **signal producer**, represented by the [SignalProducer][] class, creates 
+Signals and performs side effects.
+
+They can be used to represent operations or tasks, like network 
+requests, where each invocation of `start()` will create a new underlying 
+operation. The produced signal is returned to the caller, who can observe
+the result of the task by observing the signal.
+
+Because of the behavior of `start()`, different Signals created from the 
+producer may see a different version of events the events may arrive in a 
```

Comment:
Missing a comma - may see a different version of events**,** the events may arrive in a